Meta/CMake/flac_spec_tests.cmake

```diff
@@ -0,0 +1,21 @@
+if(INCLUDE_FLAC_SPEC_TESTS)
+    if (CMAKE_PROJECT_NAME STREQUAL "SerenityOS")
+        set(SOURCE_DIR "${SerenityOS_SOURCE_DIR}")
+    else()
+        set(SOURCE_DIR "${SERENITY_PROJECT_ROOT}")
+    endif()
+    set(FLAC_SPEC_TEST_GZ_URL https://github.com/ietf-wg-cellar/flac-test-files/archive/refs/heads/main.tar.gz)
+    set(FLAC_SPEC_TEST_GZ_PATH ${CMAKE_BINARY_DIR}/flac-spec-testsuite.tar.gz)
+    set(FLAC_SPEC_TEST_PATH ${SOURCE_DIR}/Tests/LibAudio/FLAC/SpecTests)
```

Comment:
Don't do this. Extract into the binary directory instead of the source tree, and then we don't need the git ignore entry